### PR TITLE
Moved from mackerel-agent-plugins

### PR DESCRIPTION
**Pull request by shibayu36**
This pull request does

- Separated mackerel-plugin-aws-ec2 from mackerel-agent-plugins
- Implemented the specification of `mkr plugin install`
    - Make darwin/linux binary because this plugin doesn't depend on CPU architecuture

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v0.0.1 (2017-11-07)
+
+* Moved from https://github.com/mackerelio/mackerel-agent-plugins
```
Comment (astj) [inline CHANGELOG.md]: It's better to mention which revision moved from.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v0.0.1 (2017-11-07)
```
Comment (astj) [inline CHANGELOG.md]: I'm not sure v0.0.1 is proper version. imo well-tested and/or widely-used plugins can start from v1.0.0.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# How to release
```
Comment (astj) [inline RELEASE.md:1]: It looks like this file will be copied to many repositories. Is there a more healthy way to handle this file?
(Put master file on somewhere and mention to the file in each repository?)

File: .goxc.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        "rmbin"
+    ],
+    "TaskSettings": {
+        "archive-zip": {
+            "include-top-level-dir": "windows darwin linux",
+            "platforms": "windows darwin linux"
+        }
+    },
+    "Arch": "386 amd64",
+    "Os": "linux darwin windows",
```
Comment (astj) [inline .goxc.json]: I don't know whether this plugin works in Windows. I hope, but maybe never tested.

File: CHANGELOG.md (hunk 2 above)
Comment (shibayu36) [reply CHANGELOG.md]: I agree. I will change it.

File: CHANGELOG.md (hunk 1 above)
Comment (shibayu36) [reply CHANGELOG.md]: - You mean that it should refer to https://github.com/mackerelio/mackerel-agent-plugins/pull/438 ?
- That pull request has been merged yet, so there isn't the revision which can be refered...

File: .goxc.json (hunk 4 above)
Comment (shibayu36) [reply .goxc.json]: I'll check it.

File: CHANGELOG.md (hunk 1 above)
Comment (astj) [reply CHANGELOG.md]: I imagined the revision just you've copied from, but using https://github.com/mackerelio/mackerel-agent-plugins/pull/438 looks good.  After merged this p-r and the one we can mention the commit?

**Review by astj — APPROVED** [review]

**Pull request merged**